Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Draenei Ethnicity

## Changelog:
- @ercarp: Reworked the current draenei ethnicity. New one is called `draenei_ethnicity_rework` and is based on the models used in _World of Warcraft_.
- Assigned hairstyles and beards to draeneis.
- @zumbak04: Imported missing female draenei horns from WoW.
- @ercarp: Added **Man'ari Eredar** ethnicity.


# How to test:
- Check the Draenei Eredar and Man'ari Eredar ethnicities in the Ruler Designer and give feedback on what you think could be improved or changed.
- Consider the following when testing: **1)** are the generations unique enough, **2)** are the hairstyles and beards fitting, **3)** and do they (in your subjective opinion) look good? If not, try to be specific about what could be changed to fix whatever is wrong with them.

# Tasks:
- [x] Design the Draenei ethnicity. [Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Ethnicity-Guidelines). I (@zumbak04) have added horns, tails, tendrils you will need for the Draenei but we've not worked on their genes: how their faces and bodies look. That's a task to do.
- [ ] Fix a bug where you can pick 2 horns: if you generate a Draenei and then pick horns in the Horns tab.
- [x] Add Man'ari Eredars. 
- [x] Localize female Draenei horns.
- [x] Add female Draenei horns to customization.
- [x] Localize male Draenei horns.
- [x] Add male Draenei horns to customization
- [x] Fix child tendrils
- [x] Force child to have small horns
- [x] Check for errors
